torch/custom_class.h

```diff
@@ -98,23 +124,28 @@ class class_ {
   /// `torch::init<int, std::string>()` would register a two-argument constructor
   /// taking an `int` and a `std::string` as argument.
   template <typename... Types>
-  class_& def(detail::types<void, Types...>, std::string doc_string = "") { // Used in combination with
-                                               // torch::init<...>()
+  class_& def(
+      detail::types<void, Types...>,
+      std::string doc_string = "",
+      std::initializer_list<arg> default_args =
+          {}) { // Used in combination with
```

Comment:
The format here looks pretty weird. Did clang-format got fooled by the line break in comment?